backend/LexBoxApi/Services/ProjectService.cs

```diff
@@ -70,6 +70,7 @@ public async Task FinishReset(string code, Stream zipFile)
         if (project.ResetStatus != ResetStatus.InProgress) throw ProjectResetException.NotReadyForUpload(code);
         await _hgService.FinishReset(code, zipFile);
         project.ResetStatus = ResetStatus.None;
+        project.LastCommit = await _hgService.GetLastCommitTimeFromHg(project.Code, project.MigrationStatus);
```

Comment:
could this fail? we've seen hg throw errors for projects right after they're created. Could this do the same thing?